aks/application/resources.tf

```
@@ -244,3 +244,37 @@ resource "kubernetes_secret" "ghcr_auth" {
     })
   }
 }
+
+resource "azurerm_monitor_metric_alert" "container_restarts" {
```

```suggestion
resource "azurerm_monitor_metric_alert" "restarts" {
```

Just a suggestion so it matches the `name`, or the `name` could be `"${local.app_name}-container-restarts"`.

---

changed the name to ${local.app_name}-container-restarts as it was clearer on the alerts emails